File: mults_of_3_and_5/php/mults_of_3_and_5.php
<?php

class Mults_of_3_and_5 {
	public static function getSum($max) {
		return self::get3Sum($max) + self::get5Sum($max);
	}
	private static function get3Sum($max) {
		$sum=0;
		for ($i=0; $i < $max; $i+=3) { 
			$sum+=$i;
		}
		return $sum;
	}
	private static function get5Sum($max) {
		$sum=0;
		for ($i=0; $i < $max; $i+=5) { 
			if ($i % 3 != 0) {
				$sum+=$i;
			}
		}
		return $sum;
	}
}

var_dump(Mults_of_3_and_5::getSum(1000));

File: mults_of_3_and_5/java/mults_of_3_and_5.java
public class mults_of_3_and_5 {
	public static void main(String[] args) {
		System.out.println(get3Sum(Integer.parseInt(args[0]))+get5Sum(Integer.parseInt(args[0])));
	}

	private static int get3Sum(int max) {
		int total=0;
		for (int i=0; i<max; i+=3) {
			total += i;
		}
		return total;
	}

	private static int get5Sum(int max) {
		int total=0;
		for (int i=0; i<max; i+=5) {
			if (i % 3 != 0) {
				total += i;
			}
		}
		return total;
	}
}